Source/Atomic/Graphics/Material.cpp

```diff
@@ -758,7 +758,7 @@ Scene* Material::GetScene() const
 
 String Material::GetTextureUnitName(TextureUnit unit)

```

Comment:
How about adding a static getter on Material instead of exposing the raw string array?  I can do this if there are time constraints